morpholopy/morphology.py

```diff
@@ -548,20 +545,28 @@ def get_kappa_corot(
     if K == 0.0:
         return 0.0, 0.0
 
-    # np.cross does not preserve units, so we need to multiply them back in
-    angular_momentum = (
-        (mass[:, None] * np.cross(position, velocity)) * position.units * velocity.units
-    )
+    # np.cross may not preserve units, so we need to multiply them back in
```

Comment:
Apparently this depends on the version of `unyt`

Reply:
Should we add unyt >= 3 to `requirements.txt` instead of supporting both cases?

Reply:
Agreed!